src/sync/incomingSyncUtils.js

```diff
@@ -233,6 +234,10 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
       cannotBeBlank: ['name', 'startDate', 'endDate', 'periodScheduleID'],
       canBeBlank: [],
     },
+    Unit: {
+      cannotBeBlank: [],
+      canBeBlank: ['units', 'comment', 'order_number'],
```

Comment:
Similar to my comment on the schema, I think 'units' should be 'cannotBeBlank' cause the record is screwed without it.